src/components/image/image.js

```diff
@@ -85,59 +87,56 @@ export class EuiImage extends Component {
       );
     }
 
-    let optionalIcon;
+    const allowFullScreeIcon = (
+      <EuiIcon
+        type="fullScreen"
+        color={fullScreenIconColorMap[fullScreenIconColor]}
+        className="euiImage__icon"
+      />
+    );
 
-    if (allowFullScreen) {
-      optionalIcon = (
-        <EuiIcon
-          type="fullScreen"
-          color={fullScreenIconColorMap[fullScreenIconColor]}
-          className="euiImage__icon"
-        />
-      );
-    }
+    const fullScreenDisplay = (
+      <EuiOverlayMask onClick={this.closeFullScreen}>
+        <EuiFocusTrap clickOutsideDisables={true}>
+          <button
+            type="button"
+            onClick={this.closeFullScreen}
+            onKeyDown={this.onKeyDown}>
+            <figure
+              ref={node => {
+                this.figure = node;
+              }}
+              className="euiImageFullScreen">
```

Comment:
For BEM naming conventions, since `FullScreen` is actually a state, you should use a single dash like:
```suggestion
              className="euiImage-isFullScreen">
```

Reply:
Thanks for the suggestion. This class was part of the old code and I was not sure if I should change it. I updated it with the BEM naming convention and the corresponding SASS file.